content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -227,6 +227,22 @@ can happen, according to:
 - the type of controller
 - the cluster's resource capacity
 
+## Pod disruption conditions {#pod-disruption-conditions}
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+When you enable the `PodDisruptionsCondition` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+in your cluster, a dedicated Pod `DisruptionTarget` condition is added to indicate
+a disruption. Its `reason` field additionally indicates one of the following
+reasons for the Pod termination:
+- `PreemptionByKubeScheduler` (Pod preempted by kube-scheduler)
+- `DeletionByTaintManager` (Pod deleted by taint manager due to NoExecute taint)
+- `EvictionByEvictionAPI` (Pod evicted by Eviction API)
```

Comment:
link to https://kubernetes.io/docs/concepts/scheduling-eviction/api-eviction/

Reply:
Added